omou-frontend/src/reducers/initialState.js

```diff
@@ -342,8 +342,64 @@ export default {
             },
         ],
     },
+    "CalenderData": {
+        "events_in_view": [{
+            id: 1,
+            title: "6th Grade Math Placement Test Prep",
+            start: '2019-07-22T07:00',
+            end: '2019-07-22T08:00',
+            editable: true,
+            course: "math",
+            instructor: "calvin"
```

Comment:
Name instructors by instructor id's so we can find the full instructor object in the user model (i.e. calvin teacher has an id of 23, we would list 23 here). 

`instructor_id : 23`

Reply:
we do have fake courses for you to use for this. Reference [this google spreadsheet](https://docs.google.com/spreadsheets/d/1kjw--9xXK0hDznDU4GrYW0UybXr4oI71dymmLRfDC5k/edit?usp=sharing)